lib/hooks/useFacets/useFacetsQuery/useFacetsQuery.js

```diff
@@ -42,7 +43,10 @@ const useFacetsQuery = ({
       return 'search/subjects/facets';
     } else if (Object.values(browseClassificationOptions).includes(qindex)) {
       return 'search/classifications/facets';
+    } else if (Object.values(browseCallNumberOptions).includes(qindex)) {
+      return 'search/call-numbers/facets';
```

Comment:
Let's mark this PR as *BREAKING* as it involves removing existing functionality and is incompatible with the prior version due to the use of a new endpoint.